dataprovider/src/main/java/cm/aptoide/pt/dataprovider/model/v7/timeline/Article.java

```diff
@@ -1,48 +0,0 @@
-package cm.aptoide.pt.dataprovider.model.v7.timeline;
-
-import cm.aptoide.pt.dataprovider.model.v7.listapp.App;
-import com.fasterxml.jackson.annotation.JsonCreator;
-import com.fasterxml.jackson.annotation.JsonFormat;
-import com.fasterxml.jackson.annotation.JsonProperty;
-import java.util.Date;
-import java.util.List;
-
-public class Article extends Feature {
-
-  private final Publisher publisher;
```

Comment:
Publisher doesn't any other usages, so it could be deleted

Reply:
deleted